crates/chain-state/src/in_memory.rs

```diff
@@ -959,6 +960,14 @@ mod tests {
             Ok(AccountProof::new(Address::random()))
         }
 
+        fn multiproof(
+            &self,
+            _hashed_state: HashedPostState,
+            _targets: HashMap<B256, HashSet<B256>>,
+        ) -> ProviderResult<MultiProof> {
+            Ok(MultiProof::default())
```

Comment:
This impl is repeated several times, should we define it as the default implementation in the trait?

Reply:
this is okay for mock / test implementations, but it should never return default in real impls